src/com/lftechnology/java/training/bhuwan/PullRequestDemo1.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * 
+ */
+package com.lftechnology.java.training.bhuwan;
+
+/**
+ * @author bhuwan
+ */
+public class PullRequestDemo1 {
+
+    public static void main(String[] args) {
+        int a;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "a" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)